include/seqan3/alignment/configuration/detail.hpp

```diff
@@ -32,7 +32,7 @@ enum struct align_config_id : uint8_t
     parallel,     //!< ID for the seqan3::align_cfg::parallel option.
     result,       //!< ID for the \ref seqan3::align_cfg::result "result" option.
     scoring,      //!< ID for the \ref seqan3::align_cfg::scoring "scoring" option.
-    vectorise,    //!< ID for the seqan3::align_cfg::vectorise option.
+    vectorise,    //!< ID for the \ref seqan3::align_cfg::vectorise "vectorise" option.
```

Comment:
Thematically, Vectorise should be written directly under parallel. Additionally it makes sense to scoring/result configuration at the end. 
The configuration matrix below this snippet was changed beforehand? The matrix has to be adjusted accordingly.

Reply:
The members had been ordered alphabetically, and we just kept this style. I don't see any advantage of grouping, as this interface is not public to users. 

Yes, the matrix has been updated in PR #1313 last week.

Reply:
Ok I did not see it ordered alphabetically.